.github/actions/asana-log-message/action.yml

```diff
@@ -54,9 +54,13 @@ runs:
         fi
 
     - id: add-comment
-      uses: ./.github/actions/asana-add-comment
-      with:
-        access-token: ${{ inputs.access-token }}
-        task-id: ${{ steps.get-automation-subtask.outputs.asana_automation_task_id }}
-        comment: ${{ inputs.comment }}
-        template-name: ${{ inputs.template-name }}
+      shell: bash
+      env:
+        ASANA_ACCESS_TOKEN: ${{ inputs.access-token }}
+        WORKFLOW_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+      run: |
+        bundle exec fastlane run asana_add_comment \
+          task_id:"${{ steps.get-automation-subtask.outputs.asana_automation_task_id }}" \
+          comment:"${{ inputs.comment }}" \
+          workflow_url:"${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}" \
```

Comment:
this will be removed in next PR as we shouldn't set workflow_url both through input var and env var

Reply:
If passing workflow_url on the command line could be useful in any case, you could keep it as ConfigItem, but specify `env_name: "WORKFLOW_URL"`, just like we do for Asana and Github tokens.